src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-100800.mdx

```diff
@@ -0,0 +1,45 @@
+---
+subject: 'Python agent'
+releaseDate: '2025-03-27'
+version: 10.8.0
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: ['Add support for Kombu', 'Add support for mysqlclient', 'Update instrumentation for PyMySQL for versions 1.0.3 and above']
+bugs: ['Safeguard startup of agent healthchecks against exceptions', 'Fix warning message around garbage collection runtime metrics sampler']
+---
+
+## Notes
+
+This release of the Python agent adds support for 
+
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+
+## New features
+
+* Add support for Kombu
+
+  * Add support for producer and consumer message tracing in [Kombu](https://pypi.org/project/kombu/).
+  * In the case where Celery and Kombu are used together, a transaction is produced for both Celery and Kombu message consumption and Kombu consumer tracing is turned off by default. It can be enabled via the `instrumentation.kombu.consumer.enabled` setting. A new setting `instrumentation.kombu.ignore_exchanges` to ignore certain exchanges was also added and by default it ignores the celeryev queue which is a monitoring queue.
```

Comment:
```suggestion
  * In the case where Celery and Kombu are used together, a transaction is produced for both Celery and Kombu message consumption and Kombu consumer tracing is turned off by default. It can be enabled via the `instrumentation.kombu.consumer.enabled` setting. A new setting `instrumentation.kombu.ignored_exchanges` to ignore certain exchanges was also added and by default it ignores the celeryev queue which is a monitoring queue.
```